core/src/main/scala/dagr/core/execsystem/TaskManager.scala

```diff
@@ -141,7 +141,7 @@ class TaskManager(taskManagerResources: SystemResources = TaskManagerDefaults.de
   private val actualScriptsDirectory = scriptsDirectory getOrElse Io.makeTempDir("scripts")
   protected val actualLogsDirectory  = logDirectory getOrElse Io.makeTempDir("logs")
 
-  protected val taskExecutionRunner: TaskExecutionRunnerApi = new TaskExecutionRunner()
+  private val taskExecutionRunner: TaskExecutionRunnerApi = new TaskExecutionRunner()
```

Comment:
I use the `terminateTask` instead (from #196) and forgot to make it private again.